README.md

```diff
@@ -53,11 +53,20 @@ yarn install
 ### Setup environment variables
 
 ```sh
-cp ./apps/.env.defaults ./apps/.env
+cp ./apps/browser-extension-wallet/.env.defaults ./apps/browser-extension-wallet/.env
 ```
 
 Once `.env` files is created adjust it to your needs, and update `LACE_EXTENSION_KEY`
```

Comment:
This doesn't seem necessary, and it causes confusion.

@Cauterization @HarunJr I'd appreciate your thoughts.
```suggestion

```